ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -615,34 +615,33 @@ private void logImportedBlockInfo(
       final Optional<Integer> nbParallelizedTransactions) {
     final StringBuilder message = new StringBuilder();
     final int nbTransactions = block.getBody().getTransactions().size();
-    message.append("Imported #%,d  (%s)|%5d tx");
+    message.append("Imported #%,d  (%s)| %4d tx");
     final List<Object> messageArgs =
         new ArrayList<>(
             List.of(
                 block.getHeader().getNumber(), block.getHash().toShortLogString(), nbTransactions));
+    if (nbParallelizedTransactions.isPresent()) {
+      double parallelizedTxPercentage =
+          (double) (nbParallelizedTransactions.get() * 100) / nbTransactions;
+      message.append(" (%5.1f%% parallel)");
+      messageArgs.add(parallelizedTxPercentage);
+    }
     if (block.getBody().getWithdrawals().isPresent()) {
-      message.append("|%3d ws");
+      message.append("| %2d ws");
       messageArgs.add(block.getBody().getWithdrawals().get().size());
     }
     double mgasPerSec = (timeInS != 0) ? block.getHeader().getGasUsed() / (timeInS * 1_000_000) : 0;
     message.append(
-        "|%2d blobs| base fee %s| gas used %,11d (%5.1f%%)| exec time %01.3fs| mgas/s %6.2f");
+        "| %d blobs| %s base| %,10d (%5.1f%%) gas used| %01.3fs exec| %6.2f Mgas/s| %2d peers");
```

Comment:
With peerdas scheduled for Fusaka, blob counts are expected to exceed 10, so we should stick with at least two-digit formatting to maintain consistent log output.

Reply:
The same for gas used, we want to support more gas limit starting from fusaka, we are discussing about 100 and 150 million gas limit in the next months. So we still need to have 11 characters for gas used (9 digits and two commas).

Reply:
I suggest to replace `base` with `bfee`, because doesn't reflect the base fee and `exec` with `time`

Reply:
We can also replace `gas used` with `gas` to reduce more

Reply:
> We can also replace `gas used` with `gas` to reduce more

Replacing `gas used` with `gas` may cause ambiguity as in `gas used` vs `gas limit`. 🤔

Reply:
> I suggest to replace `base` with `bfee`, because `base` doesn't reflect the base fee and `exec` with `time`

The unit `s` is probably already indicative of time?